Go/Bitcoin_Price_Tracker/Readme.md

```diff
@@ -0,0 +1,4 @@
+# Bitcoin_Price_Tracker
+This is a script written in **golang** to fetch the live price of **1 BTC** in **INR**. I have used the [colly package](https://github.com/gocolly/colly) to scrape the
+webpage and search the relevant information.
+
```

Comment:
Can you please follow this [template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) ?